src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -921,4 +926,35 @@ private class ACCT_IndividualAccounts_TEST {
         system.assertEquals(con2.Salutation + ' ' + con2.FirstName + ' Family', assertAccount.Name);
 
     }
+
+    /*********************************************************************************************************
+    * @description Test Method for detaching a Contact from an already existing account 
+    * which is not the default account model.
+    */
+    @isTest
+    public static void detachFromBizAccountToAdministrative() {
+        UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c(Account_Processor__c = UTIL_Describe_API.getAdminAccRecTypeID()));
+
+        Account acct = UTIL_UnitTestData_API.getMultipleTestAccounts(1, UTIL_Describe_API.getBizAccRecTypeID())[0];
+        insert acct;
+
+        Contact con = UTIL_UnitTestData_API.getContact();
+        con.AccountId = acct.id;
+        insert con;
+
+        Contact[] insertedContacts = [SELECT Account.Name, AccountId FROM Contact where id=:con.id];
+        //the contact should be connected to the account
+        System.assertEquals(acct.id, insertedContacts[0].AccountId);
+
+        con.AccountId = NULL;
+        Test.startTest();
+        update con;
+        Test.stopTest();
+
+        Contact[] updatedContacts = [SELECT Account.Name, AccountId FROM Contact where id=:con.id];
+
+        //the account should not be the one we started with
+        System.assertNotEquals(acct.id, updatedContacts[0].AccountId);
```

Comment:
We should also assert that it's not null, and possibly assert that the account's RecordTypeId = UTIL_Describe_API.getAdminAccRecTypeID().

Reply:
Hmm we are checking if the ids are equal so no need for assert that it is not null, right? The contact/account is not deleted. Same reason for recordtype, the ids are same any way and there is no code that changes the recordtype.

Reply:
oops discard this please, I was looking at the other method.